src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -1,46 +1,121 @@
 package seedu.duke.manager;
 
+import java.util.Arrays;
+import java.util.List;
 import seedu.duke.entities.Dish;
 import seedu.duke.entities.Order;
+import seedu.duke.loggers.MainLogger;
 
 import java.util.ArrayList;
 
-
-public class OrderManager {
-    private final ArrayList<Order> orders;
+/**
+ * OrderManager.
+ */
+public class OrderManager extends Manager {
+    private static OrderManager singleton = null;
+    private List<Order> orders;
     private final DishManager dishManager = DishManager.getInstance();
```

Comment:
i think would be better to have dishManager as a attribute of the order controller, rather than be embedded in orderManager.

Reply:
ok